config/initializers/admin.rb

```diff
@@ -0,0 +1,9 @@
+require "rollbar/middleware/rack"
+
+require_relative "../../lib/restrict_admin_by_ip_middleware"
+
+Rails.application.configure do
+  allowed_ips = ENV.fetch("ADMIN_ALLOWED_IPS").split(",")
+
+  config.middleware.insert_after Rollbar::Middleware::Rails::RollbarMiddleware, RestrictAdminByIpMiddleware, allowed_ips
```

Comment:
Can we skip this middleware completely in dev?

Reply:
I'd prefer not to. Let's keep dev as close to production as possible. This also makes it easy to test the behaviour locally by removing the env var.